cmd/eudico/mirvalidator/utils.go

```diff
@@ -176,6 +177,38 @@ func newLp2pHost(dir string) (host.Host, error) {
 	)
 }
 
+func getLibP2PHost(dir string) (host.Host, error) {
```

Comment:
Why do we need a new function? We shouldn't need to change the function we had before right? Isn't this duplicating code?